otterdog/eclipse-sisu.jsonnet

```diff
@@ -68,7 +68,6 @@ orgs.newOrg('eclipse-sisu') {
       description: "Sisu Mojos",
       gh_pages_build_type: "legacy",
       gh_pages_source_branch: "master",
-      gh_pages_source_path: "/docs",
```

Comment:
the repo is archived already so doing there any change will not make any difference, also I am not sure if the change is even intended?

Reply:
Good catch, this change was not intended, fixed in https://github.com/eclipse-sisu/.eclipsefdn/pull/4/commits/a65f0bd86b1bf5b885dbda851977a85f5cba958b.